Reflections/Reflections_Week_2.md

```diff
@@ -0,0 +1,16 @@
+1. This week I have accomplished a few things.
+- I was able to move my distance slider to a new screen with more filters such as filtering by price and whether or not the business has to be open when searched.
+- Then I was able to finally get the "Get Directions" button to work from the results menu to where it would open Google Maps and give directions from the user's location to the restaurant.
+
+2. The biggest challenge that I faced was trying to get the "Get Directions" button to work because I had to do some research on Google Maps URLs. When I first started I tried to pull the latitude and longitude of the chosen restaurant and display that in the Google Maps URL, however Yelp stores their Lat. and Long. as decimal values and I couldn't figure out how to convert them to string in block code. So then I tried using the variables lat. and long. from my code but I realized that only gave you the user's location. So that's when it hit me to use the user's location and looking at the link Dr. Nakazawa sent me way earlier in Slack I was able to figure out the basic formula for getting the URL. I have a simpliefied version of it because I couldn't figure out URL encoding in block code (Ex. Converting Frisch's Big Boy to Frisch%27s+Big+Boy).
+
+3. Now that I feel as if I have the easier stuff done I plan on implementing some new features.
+- The spinning wheel fianlly. I looked into it and I feel as if this could probably get this done in an 2-4 hours because of testing it out. I have the color wheel finished and the arrow that I plan to use is actually a slice of pizza to make it food related. Like I did with the slider, I will build the spinning wheel in a separate thunkable project just to test it without messing up my project that I have so far.
+- Work on the design of the app. As of right now my app works and does what if needs to but doesn't look all that great, it's really plain. I plan on reworking the logo to make it a transparent background, that way when I change the screens background for the app there won't be a white block on the screen. I then need to work on a color scheme (Red and Black - Roulette Wheel) and change the colors of my buttons, text, and background to reflect this. All in all I think this could be accomplished in an hour or two.
+- Now that get my app closer to being finished I want to do some user testing and get feedback. I plan to tell my friends and family about what I have created and show them my app from Thunkable. I plan on creating a survey with consent forms to use their data from the user testing, taking notes on everyone's feedback, figure out what to change and keep the same. This probably wouldn't get done in a weeks time because I need to implement the other features first and set up appointments for the test users. (1-2 weeks)

```

Comment:
Really great idea! It will be specially useful to work on this for the upcoming assignment on Usability Testing!